lnschema_core/models.py

```diff
@@ -773,12 +773,52 @@ class ParamManagerRun(ParamManager):
 class HasFeatures:
     """Base class linking features, in particular, for :class:`~lamindb.Artifact` & :class:`~lamindb.Collection`."""
 
-    features = FeatureManager
-    """Feature manager. :class:`~lamindb.core.FeatureManager`"""
+    features: FeatureManager = FeatureManager  # type: ignore
+    """Feature manager.
+
+    Features denote dataset dimensions, i.e., the variables that measure labels
```

Comment:
Screenshot is from here.

https://laminlabs.slack.com/archives/C04FPE8V01W/p1719833212865679?thread_ts=1719450186.839499&cid=C04FPE8V01W